components/ButtonBar/src/private/MDCAppBarButtonBarBuilder.m

```diff
@@ -319,6 +313,10 @@ - (void)layoutSubviews {
   }
 }
 
+- (void)setTitleFont:(nullable UIFont *)font forState:(UIControlState)state {
```

Comment:
Do we need this implementation if it only calls super?

Reply:
We do, not sure why, added investigate TODO. #2850